cmd/aries-agent-mobile/Makefile

```diff
@@ -58,21 +73,31 @@ dependencies:
 	$(GOMOBILE_CMD) init -v
 
 .PHONY: bindings
-bindings: bindings-android bindings-ios
+bindings: clean-build bindings-android bindings-ios
 
 .PHONY: bindings-android
 bindings-android:
 	@echo '----------------------------------------'
 	@echo '   gomobile: Creating Android bindings  '
 	@echo '----------------------------------------'
 	@mkdir -p $(ANDROID_BUILD_DIR)
-	$(GOMOBILE_CMD) bind -v -target=$(ANDROID_TARGET) -o=$(ANDROID_BUILD_DIR)/$(ANDROID_OUTPUT_FILE_NAME) $(MOBILE_BINDINGS_ROOT_PACKAGE)
+	$(GOMOBILE_CMD) bind -v -target=$(ANDROID_TARGET) -javapkg=$(ANDROID_JAVA_PKG) -o=$(ANDROID_BUILD_DIR)/$(ANDROID_OUTPUT_FILE_NAME) $(MOBILE_BUILD_PACKAGES)
 
 .PHONY: bindings-ios
 bindings-ios:
 	@echo '----------------------------------------'
 	@echo '   gomobile: Creating iOS bindings      '
 	@echo '----------------------------------------'
+ifeq ($(IS_MACOS),true)
 	@mkdir -p $(IOS_BUILD_DIR)
-	$(GOMOBILE_CMD) bind -v -target=$(IOS_TARGET) -o=$(IOS_BUILD_DIR)/$(IOS_OUTPUT_FILE_NAME) $(MOBILE_BINDINGS_ROOT_PACKAGE)
+	$(GOMOBILE_CMD) bind -v -target=$(IOS_TARGET) -o=$(IOS_BUILD_DIR)/$(IOS_OUTPUT_FILE_NAME) $(MOBILE_BUILD_PACKAGES)
+else
+	@echo 'Skipping iOS bindings. Only available for Darwin operating systems'
```

Comment:
@llorllale iOS bindings require Darwin i.e macOS. Apple [doesn't allow](https://stackoverflow.com/a/2406839) Windows or Linux users to build iOS or Mac software. I will make it more explicit in the README that only macOS users will be able to create iOS bindings.

Reply:
this is in response to: https://github.com/hyperledger/aries-framework-go/pull/1955#issuecomment-653537494

Reply:
for the second part of that comment, i added in the requirements section of the README that Android Studio should be installed. I can make it more explicit. The android sdk can be installed without Android Studio but it's more of a hassle to get other things working like emulators and the rest.

the speed of the bindings is what's currently available. it pulls packages from the internet so a slow connection speed and low bandwidth could make the process seem slow. these were my results:

```bash
$ time make bindings-android
.....
real	1m7.049s
user	1m42.522s
sys	0m45.966s
```

Reply:
@ToJen I think my https://github.com/hyperledger/aries-framework-go/pull/1955#issuecomment-653537494 may be misinterpreted as a critique of this PR, or otherwise an account of "points against" this change. It is not. It's just my notes. :)

Reply:
thank you, i got that. i wanted to respond just for documentation purposes. i'll use them as suggestions for the README  file 👍